specification/cpim/resource-manager/readme.python.md

```diff
@@ -53,5 +53,6 @@ These settings apply only when `--tag=package-2019-01-01-preview --python` is sp
 
 ```yaml $(tag) =='package-2019-01-01-preview'&& $(python)
 namespace: $(python-base-namespace).v2019_01_01_preview
-output-folder: $(python-sdks-folder)/azureadexternalidentities/azure-mgmt-azureadexternalidentities/azure/mgmt/azureadexternalidentities/v2021_04_01
+output-folder: $(python-sdks-folder)/azureadexternalidentities/azure-mgmt-azureadexternalidentities/azure/mgmt/azureadexternalidentities/v2019_01_01_preview
```

Comment:
(1)`python-base-folder` is not used, so we need to delete it
(2) `python-base-namespace` is not defined before, so we need to change all the "$(python-base-namespace)" to `azure.mgmt.azureadexternalidentities`